test/test_coordinates_photo.py

```diff
@@ -8,6 +8,23 @@
 # Import the functions to test
 import coordinates_photo as cp
 
+class TestCameraCalibrationWithAstrometryNet(unittest.TestCase):
+    def test_calibrate_camera(self):
+        photo = "images/test/test_12_mars_21h46mn36s.jpg"
+        reference_pixel, reference_point, transformation_matrix = cp.calibrate_camera(photo)
+        self.assertEqual(reference_pixel, (955.153951009, 437.333496094))
+        self.assertEqual(reference_point, (79.8535963865, 10.0878946534))
+        self.assertEqual(transformation_matrix, ([-0.0358029908004, -0.027435406908],
+                                                 [0.0270064129609, -0.034174545714]))
+
+class TestCameraCalibrationWithoutAstrometryNet(unittest.TestCase):
```

Comment:
maybe the name of this test is not exactly clear but hey i'm just tryna find comments to add